src/main/java/com/github/shyiko/mysql/binlog/event/deserialization/AbstractRowsEventDataDeserializer.java

```diff
@@ -376,7 +383,7 @@ private static int extractBits(long value, int bitOffset, int numberOfBits, int
     /**
      * see mysql/strings/decimal.c
      */
-    private static BigDecimal asBigDecimal(int precision, int scale, byte[] value) {
+    public static BigDecimal asBigDecimal(int precision, int scale, byte[] value) {
```

Comment:
Yeah, it could probably be moved to ByteArrayInputStream but don't worry about. I can do that (or something entirely different) as part of #107.